### PR TITLE
Prepare the 2.20.4 release.

### PR DESCRIPTION
**Pull request by jsirois**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 2.20.4
+
+This release fixes a ug where `--pip-log <path>`, used multiple times
```
Comment (benjyw) [inline CHANGES.md:5]:
```suggestion
This release fixes a bug where `--pip-log <path>`, used multiple times
```

**Review by benjyw — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (jsirois) [reply CHANGES.md:5]: I'm having a hell of a time getting CI green today. I may skip this until post merge and just manually edit the Releases / Announcements pages post-release. We'll see.

Comment (benjyw) [reply CHANGES.md:5]: I don't think it matters at all. Fine to fix later or never.

Comment (jsirois) [reply CHANGES.md]: Fixed. I had to add 2 new fixes to the change log, so there was a good moment to do this.

**Pull request merged**